deploy/templates/notificationtemplates/appstudio/userprovisioned/notification.html

```diff
@@ -57,7 +57,7 @@
   </p>
 
   <p>
-    This project is still in its early days, so we would love your feedback! To interact with our team, you can join the DevNation Slack workspace here: https://dn.dev/DevNationSlack, and then join our #software-supply-chain-security channel.
+    This project is still in its early days, so we would love your feedback! To interact with our team, you can join the DevNation Slack workspace here: https://dn.dev/slack, and then join our #software-supply-chain-security channel.
```

Comment:
```suggestion
    This project is still in its early days, so we would love your feedback! To interact with our team, you can join the DevNation Slack workspace here: https://dn.dev/slack, and then join our [#software-supply-chain-security](https://rhdevnation.slack.com/channels/software-supply-chain-security) channel.
```

Reply:
> [#software-supply-chain-security](https://rhdevnation.slack.com/channels/software-supply-chain-security) 

@mattreid I suppose you are trying to add a hyperlink in the body of the email with [] and (), however mailgun doesn't support hyperlinks out of the box. So this change wouldn't render it as a hyperlink in user's inbox. I suggest we keep the channel in plain text as is.